package.json

```diff
@@ -165,4 +165,4 @@
   "resolutions": {
     "react-error-overlay": "6.0.9"
   }
-}
\ No newline at end of file
+}
```

Comment:
người thì thêm người thì xoá :pepe_angry:

Reply:
should unified dis

Reply:
lint auto